sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalogBase.scala

```diff
@@ -82,6 +81,8 @@ class CosmosCatalogBase
                 this.sparkSession,
                 viewRepositoryConfig.metaDataPath.get))
         }
+
+        this.clientManager = CosmosClientManagerFactory.getCosmosClientManager(config)
```

Comment:
todo: failure out error handling, for example 404. when using management API, if the to be deleted database account does not exists, no exceptions will be thrown

Reply:
updated